### PR TITLE
Feature - Programme detail

### PR DESCRIPTION
**Pull request by joedha8**
Refer to issues [10](https://github.com/AuthScureDevelopment/doc-ext-pointsnet/issues/10)

### CODE REVIEW

**Comment by joedha8** [comment]
image

File: source/includes/_graphqlApi.md [hunk 1]
```diff
@@ -2,6 +2,59 @@
 
 Graphql API
 
+## Get Detail Program
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: This text is changed to `Mendapatkan detail programme`

File: source/includes/_graphqlApi.md [hunk 2]
```diff
@@ -2,6 +2,59 @@
...
+```scheme
+  programme_detail(
+    id: $id
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: $id changed to `355675f5-1232-455a-88be-88317534a639`

File: source/includes/_graphqlApi.md [hunk 3]
```diff
@@ -2,6 +2,59 @@
...
+    "is_bank": false,
+    "is_credit_card": false,
+    "description": "<p>Bank BCA bagi bagi promo point</p>",
+    "theme_id": "729033a7-2ddb-4df6-bf89-8495b6337d7a"
+  }
+```
+
+Query ini berfungsi untuk mendapatkan detail dari program yang dipilih
+
+### Request
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: This text is changed to `Arguments`

File: source/includes/_graphqlApi.md [hunk 4]
```diff
@@ -2,6 +2,59 @@
...
+------|-----------|--------|----------
+id | UUID | a02fd8bb-b6c0-4bbe-bcbb-9045a2b974ea | Ini adalah ID yang akan dikirim untuk mendapatkan detail programnya
+
+### Response
+
+Field | Tipe Data | Contoh | Deskripsi
+------|-----------|--------|----------
+id | UUID | a02fd8bb-b6c0-4bbe-bcbb-9045a2b974ea | Ini adalah ID dari detail program yang dipilih
+name | String | BCA BAGI BAGI | Ini adalah nama program dari detail program yang dipilih
+logo | URL | https://2.bp.blogspot.com/-_BITDWSaNos/WKggIVMUczI/AAAAAAAAB4g/d5-te8J3Ahos89_RQf0UkbTXKOQVQHDRwCPcB/s1600/Logo%2BBank%2BBCA_PNG.png | Ini adalah url image logo dari
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: url is changed to `https://your-site.com/logo.png`

File: source/includes/_graphqlApi.md [hunk 5]
```diff
@@ -2,6 +2,59 @@
...
+
+Query ini berfungsi untuk mendapatkan detail dari program yang dipilih
+
+### Request
+
+Field | Tipe Data | Contoh | Deskripsi
+------|-----------|--------|----------
+id | UUID | a02fd8bb-b6c0-4bbe-bcbb-9045a2b974ea | Ini adalah ID yang akan dikirim untuk mendapatkan detail programnya
+
+### Response
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: Changed to `Fields`

File: source/includes/_graphqlApi.md [hunk 6]
```diff
@@ -49,11 +49,29 @@ id | uuid | a02fd8bb-b6c0-4bbe-bcbb-9045a2b974ea | Ini adalah ID dari detail pro
 name | string | BCA BAGI BAGI | Ini adalah nama program dari detail program yang dipilih
 logo | string | https://your-site.com/logo.png | Ini adalah url image logo dari program yang dipilih
 bank_id | uuid | 244f9582-b00c-11e8-96f8-529269fb1459 | Ini adalah ID bank yang mengadakan program ini
-card_number_format | string | XXXX XXXX XXXX XXXX | Ini adalah format dari nomor kartu yang diizinkan
+card_number _format | string | XXXX XXXX XXXX XXXX | Ini adalah format dari nomor kartu yang diizinkan
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: card_number_format  gak pake spasi

File: source/includes/_graphqlApi.md [hunk 7]
```diff
@@ -49,11 +49,29 @@ id | uuid | a02fd8bb-b6c0-4bbe-bcbb-9045a2b974ea | Ini adalah ID dari detail pro
...
+issuer_currency _code | string | HKD | Ini adalah kode mata uang penerbit
+issuer_currency _exchange | string | IDR | Ini adalah nilai tukar mata uang penerbit
+point_discount _rate_percentage | int | 20 | Ini adalah nilai presentase point dikon
+min_redeem _percentage | int | 30 | Ini adalah presentase minimal untuk menebus
+facilitation_fee | int | 10 | Ini adalah biaya fasilitas
+point_cancel_fee | int | 10 | Ini adalah biaya membatalkan point
+currency _cancel_fee | string | HKD | Ini adalah mata uang biaya pembatalan
+is_active | boolean | true | Ini adalah status dari detail program, apakah aktif atau tidak
+loyalty_code | string | asdfg | Ini adalah kode loyalty
+created_at | date | 27-06-2018 | Ini adalah tanggal pembuatan dari detail program
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: ini di ganti `timestampz`

File: source/includes/_graphqlApi.md [hunk 8]
```diff
@@ -49,11 +49,29 @@ id | uuid | a02fd8bb-b6c0-4bbe-bcbb-9045a2b974ea | Ini adalah ID dari detail pro
...
+created_by | string | admin | Ini adalah pembuat detail program
+updated_at | date | 30-06-2018 | Ini adalah tanggal update dari detail program
```
Comment (jiharal) [inline source/includes/_graphqlApi.md]: di ganti pake `timestampz`

**Pull request merged**